### DISCUSSION TITLE
Sending redux store generated on server to client and populating the store on the client directly.?

### DISCUSSION THREAD

**Discussion by vaibhavkaul69** [vercel/next.js#43156] [Help]
So I am working on a NextJS application, wherein I want to fetch some data on the server, dispatch it to the Redux store, hydrate the HTML and send the HTML page + Pre-populated redux store back to the client.

I also want that the Redux store which came from the server should populate the store on the client directly instead of reinitializing the store. Is there a way to do it.?

I am using  Next 13.0.0.

**Comment by pangxiaolong** [comment]
Now I can't use reduxjs/toolkit in my application because it's not allowed in server components
